vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -363,7 +368,31 @@ void FlyoutViewManager::SetLayoutProps(
     float left,
     float top,
     float width,
-    float height) {}
+    float height) {
+  auto *pFlyoutShadowNode = static_cast<FlyoutShadowNode *>(&nodeToUpdate);
+
+  if (pFlyoutShadowNode->GetFlyout()) {
```

Comment:
nit: `if (auto flyout = pFlyoutShadowNode->GetFlyout())`
You don't need to split it into two lines